.circleci/config.yml

```diff
@@ -115,25 +115,25 @@ references:
   restore-npm-cache: &restore-npm-cache
     name: 'Restore npm cache'
     keys:
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v3-npm-modules-{{ checksum ".nvmrc" }}-{{ checksum "package-lock.json" }}
-      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-v3-npm-modules-{{ checksum ".nvmrc" }}
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-node-modules-{{ checksum ".nvmrc" }}-{{ checksum "yarn.lock" }}
+      - v{{ .Environment.GLOBAL_CACHE_PREFIX }}-node-modules-{{ checksum ".nvmrc" }}
 
   npm-install: &npm-install
     environment:
       PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
-    name: Install npm dependencies
-    command: npm ci
+    name: Install dependencies
+    command: yarn install --frozen-lockfile
```

Comment:
Just heads up (emphasis mine):

> If the --immutable option is set, Yarn will abort with an error exit code if anything in the install artifacts (yarn.lock, .pnp.js, ...) was to be modified. For backward compatibility we offer an **alias under the name of --frozen-lockfile, but it will be removed in a later release.**

https://yarnpkg.com/cli/install#details

Reply:
Thanks for the heads up! I think it'd be ideal to stick with v1 until a few of the teething problems with v2 are sorted out but that's good to keep in mind for v2!